### PR TITLE
Add attachment preview to review interface.

### PR DESCRIPTION
**Pull request by jmcarp**
Part of https://github.com/eregs/notice-and-comment/issues/61.

The gif doesn't look like much, since it doesn't record the file actually downloading, but that's what's happening.

1rzzzpvci0

### CODE REVIEW

File: regulations/views/comment.py [hunk 1]
```diff
@@ -43,9 +43,34 @@ def upload_proxy(request):
...
+    disposition = 'attachment; filename="{}"'.format(name)
+    url = s3.generate_presigned_url(
+        ClientMethod='get_object',
+        Params={
+            'ResponseContentDisposition': disposition,
+            'Bucket': settings.ATTACHMENT_BUCKET,
+            'Key': key,
+        },
+    )
+    return JsonResponse({'url': url})
```
Comment (cmc333333) [inline regulations/views/comment.py]: This is a tad concerning as we're no longer tying the ability to preview a document with the "session" that created it. I'm pretty sure it's not actually an issue as, our "key"s are ~300bits of randomness, but I wonder if there's not a simple way to tie this a bit tighter. For example, would it make sense to issue both a preview _and_ upload URL when the user first requests to upload the file? Presumably the URLs have a timeout, which would make that less useful.

File: regulations/urls.py [hunk 2]
```diff
@@ -45,6 +45,7 @@
     # about page
     url(r'^about$', about, name='about'),
     url(r'^comments/attachment$', comment.upload_proxy),
+    url(r'^comments/attachment/preview$', comment.download_proxy),
```
Comment (cmc333333) [inline regulations/urls.py]: What do you think about making this a RESTful url, instead? Something like `comments/attachment/<key>`

File: regulations/views/comment.py (hunk 1 above)
Comment (jmcarp) [inline regulations/views/comment.py]: I like the idea of sending back the GET and PUT URLs in the same response. Expiration times are just epoch times, so we can send back a URL that lasts for a long time--like a year, or the commenting deadline for the current proposal. Does that sound right?

Comment (cmc333333) [inline regulations/views/comment.py]: Sounds great. Let's do 90 days and we'll see if we can't tighten it to the comment deadline when that data's available.

**Comment by cmc333333** [comment]
Nice. I think this is simpler, to read, too.

**Pull request merged**